src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -63,15 +63,19 @@ const isSortEqualToField = (
   return sort.field === sortKey;
 };
 
-const fieldToSort = (field: Field, tableMeta: MetaType | undefined): Sort | undefined => {
+export const fieldToSort = (
```

Comment:
Should EventView expose a method to add a sort based on a field? It might be a simpler interface to use than having to make a field, convert the field, and then sort the eventview.

Reply:
Done in https://github.com/getsentry/sentry/pull/19230/commits/0a12c5d55d18ec0bc7c975cf83021bd8e1b4654d and https://github.com/getsentry/sentry/pull/19230/commits/db324131cc842080055e5781dd7b48f926b713bb